src/cart/src/cart/crt_swim.c

```diff
@@ -214,7 +225,7 @@ static int crt_swim_send_message(struct swim_context *ctx, swim_id_t to,
 	}
 
 	if (opc_idx == 0) { /* set timeout for one way RPC only */
-		rc = crt_req_set_timeout(rpc_req, CRT_SWIM_RPC_TIMEOUT);
+		rc = crt_req_set_timeout(rpc_req, crt_swim_rpc_timeout);
```

Comment:
Are we parsing the env variable for each SWIM RPC that is being sent?

Reply:
No, ENV is parsed only once when SWIM is initialized.